### PR TITLE
Add help text to maps

### PR DESCRIPTION
**Pull request by cdccollins**
Accessibility: maps should have text to help users who may have difficulty interacting with them

### CODE REVIEW

File: app/views/shared/_alternative_map.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<p class="govuk-body">
+  If you need this information in an alternative format, contact your case officer on <%= @local_authority["feedback_email"] %><%= @local_authority["telephone_number"].present? ? " or by telephone at #{@local_authority["telephone_number"]}." : "." %>
```
Comment (benjamineskola) [inline app/views/shared/_alternative_map.html.erb:2]: What happens here if the LA hasn't set their contact details?

Comment (cdccollins) [reply app/views/shared/_alternative_map.html.erb:2]: I think the email is mandatory, and the phone number not?

Comment (cdccollins) [reply app/views/shared/_alternative_map.html.erb:2]: But I should check

Comment (benjamineskola) [reply app/views/shared/_alternative_map.html.erb:2]: Somehow I managed to overlook that there's a ternary here checking for that precise situation. 🤦🏻  ignore me.

**Review by benjamineskola — APPROVED** [review]

File: app/views/shared/_alternative_map.html.erb (hunk 1 above)
Comment (cdccollins) [reply app/views/shared/_alternative_map.html.erb:2]: No i've checked and email isn't mandatory either! Will fix 😄

**Pull request merged**